.github/workflows/pull-request.yaml

```diff
@@ -92,7 +92,7 @@ jobs:
     # Check out `test-all.yaml` for more ideas on organizing these.
     if: always()
     needs:
```

Comment:
I think it would be nice to include the new `check-links-book` job under `check-ok-to-merge.needs` ... however I guess then we could no longer use `on.pull_request_paths` in the new workflow file? And would probably have to do something like [what's described in this blog post](https://www.meziantou.net/executing-github-actions-jobs-or-steps-only-when-specific-files-change.htm)?

Reply:
Yes, unfortunately I'm not sure it's easy to do this...

Reply:
Just saw what you did here — nice! I had seen this action before, but hadn't tried it out yet. Looks good; we could think about extending to more places if it turns out well.